src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -150,9 +153,22 @@ extension RiskProvider: RiskProviding {
 	}
 	#else
 	private func _requestRiskLevel(userInitiated: Bool, completion: Completion? = nil) {
-		let group = DispatchGroup()
+		func completeOnTargetQueue(risk: Risk?) {
+			targetQueue.async {
+				completion?(risk)
+			}
+		}
 
 		var summaries: Summaries?
```

Comment:
move these stand alone variables to the top of the class. It's just so much easier than searching for them somewhere in the middle of the code. Bonus points if you add a thoughtful `// MARC: - ` as well (hope you see what I did here)

Reply:
`// MARC: - They are not standalone` 😁

Reply:
ah. Now I see. Sorry, the github review window confused me with the method structure. Thought it was a class variable basically.